cluster/addons/volumesnapshots/volume-snapshot-controller/volume-snapshot-controller-deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       serviceAccount: volume-snapshot-controller
       containers:
         - name: volume-snapshot-controller
-          image: k8s.gcr.io/sig-storage/snapshot-controller:v4.0.0
+          image: k8s.gcr.io/sig-storage/snapshot-controller:v6.0.1
```

Comment:
We should wait until other sidecars are released before updating this.

Reply:
@xing-yang considering this is on the master and provisioner sidecar aleady updated to this version in master, do we have to wait for its release?